clinica/iotools/converters/ukb_to_bids/ukb_to_bids_cli.py

```diff
@@ -27,7 +27,7 @@ def cli(
     check_dcm2niix()
 
     convert_images(dataset_directory, bids_directory, clinical_data_directory)
-    _write_bidsignore(bids_directory)
+    _write_bidsignore(str(bids_directory))
```

Comment:
I'm not sure to understand why you need this casting. `_write_bidsignore` expects either a string or a Path, and will cast it's input to a Path anyway.

Reply:
`bids_directory` is declared `PathLike`, so neither a string nor a path. Since `_write_bidsignore` wants a concrete type (`str | Path`), we need to cast explicitly to satsify static type checking.